ODS AMI build failing due to go missing openshift api dependency

**Describe the bug**
ODS AMI build failing due to go missing openshift api dependency

**AMI build log file error**
amazon-ebs: go: github.com/openshift/api@v3.9.0+incompatible: reading github.com/openshift/api/go.mod at revision v3.9.0: unknown revision v3.9.0ESC[0m
ESC[1;31m==> amazon-ebs: make: *** [prep-verify] Error 1ESC[0m
2021/01/10 01:08:26 packer-builder-amazon-ebs plugin: [ERROR] Remote command exited with '2': /tmp/script_8755.sh  --branch 3.x

**Proposed solution**
Apply this change to branch 3.x: https://github.com/opendevstack/ods-core/commit/9c79d7c0265d0d848e585c4a4e74d97e87b40e02

--- a/tests/go.mod
+++ b/tests/go.mod
@@ -13,7 +13,7 @@ require (
 	github.com/jstemmer/go-junit-report v0.9.1 // indirect
 	github.com/kr/pretty v0.2.0 // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
-	github.com/openshift/api v3.9.0+incompatible
+	github.com/openshift/api v0.0.0-20180801171038-322a19404e37
 	github.com/openshift/client-go v3.9.0+incompatible
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/pkg/errors v0.8.1 // indirect

